### PR TITLE
Remove code intel latencies from pings

### PR DESCRIPTION
**Pull request by ebrodymoore**
Removing as a result of [this conversation](https://sourcegraph.slack.com/archives/CHXHX7XAS/p1598384458079100)

### CODE REVIEW

**Comment by ebrodymoore** [comment]
@dadlerj let me know how we should test this. I spun up an instance and the latencies are no longer there, but would you recommend anything else?

New:

Screen Shot 2020-09-15 at 2 32 21 PM

What it used to look like (more or less)

Screen Shot 2020-09-15 at 2 33 30 PM

File: doc/admin/pings.md [hunk 1]
```diff
@@ -33,7 +33,6 @@ By default, Sourcegraph also aggregates usage and performance metrics for some p
   - Search filters used (e.g. "type:", "repo:", "file:", "lang:", etc.)
 - Aggregate daily, weekly, and monthly latencies (in ms) of code intelligence events (e.g., hover tooltips) and search queries
 - Aggregate daily, weekly, and monthly counts of:
-  - Code intelligence events (e.g., hover tooltips)
```
Comment (dadlerj) [inline doc/admin/pings.md:36]: This seems to be the wrong bullet

File: cmd/frontend/types/types.go [hunk 2]
```diff
@@ -232,16 +232,6 @@ type CodeIntelEventCategoryStatistics struct {
 type CodeIntelEventStatistics struct {
 	UsersCount     int32
 	EventsCount    *int32
-	EventLatencies *CodeIntelEventLatencies
-}
-
-// NOTE: DO NOT alter this struct without making a symmetric change
-// to the updatecheck handler. This struct is marshalled and sent to
-// BigQuery, which requires the input match its schema exactly.
-type CodeIntelEventLatencies struct {
```
Comment (dadlerj) [inline cmd/frontend/types/types.go:241]: Given the code comment above, I don't see any changes in the updatecheck handler... Do we have a type there that needs to be changed as well?

File: doc/admin/pings.md (hunk 1 above)
Comment (efritz) [inline doc/admin/pings.md:36]: Did you mean to delete `Aggregate daily, weekly, and monthly latencies (in ms) of code intelligence events (e.g., hover tooltips) and search queries`?

File: web/src/site-admin/SiteAdminPingsPage.tsx [hunk 3]
```diff
@@ -92,10 +92,6 @@ export class SiteAdminPingsPage extends React.Component<Props, State> {
                             <li>Search filters used (e.g. "type:", "repo:", "file:", "lang:", etc.)</li>
                         </ul>
                     </li>
-                    <li>
```
Comment (efritz) [inline web/src/site-admin/SiteAdminPingsPage.tsx:95]: Same

**Review by efritz — APPROVED** [review]
See comments, LGTM otherwise

File: cmd/frontend/internal/usagestats/aggregated.go [hunk 4]
```diff
@@ -124,32 +124,20 @@ func insertCodeIntelEventStatistics(event types.AggregatedEvent, statistics *typ
 		return
 	}
 
-	makeLatencies := func(values []float64) *types.CodeIntelEventLatencies {
```
Comment (dadlerj) [inline cmd/frontend/internal/usagestats/aggregated.go:127]: Looks like the argument here (`values`) is coming from somewhere else (I assume it's being generated in the `db` package)... assuming it's not being used anywhere else, we should go ahead and remove it where it's generated as well. Do you feel like you can interpret that code to do this?

**Review by dadlerj — COMMENTED** [review]
Thanks! A few comments below.

File: doc/admin/pings.md (hunk 1 above)
Comment (ebrodymoore) [reply doc/admin/pings.md:36]: I sure did

File: cmd/frontend/types/types.go (hunk 2 above)
Comment (ebrodymoore) [reply cmd/frontend/types/types.go:241]: I don't believe so. The handler only takes in CodeIntelUsage, and there's nothing in the file that dictates what should be in CodeIntelUsage. So by just subtracting fields within CodeIntelUsage it shouldn't have an impact on the handler in the same way taking out the entire CodeIntelUsage would. Am I interpreting that correctly?

Comment (efritz) [reply cmd/frontend/types/types.go:241]: This should be safe. Deletions are safe in this content - glad the warning worked though as additions/mutations wouldn't be :)

File: cmd/frontend/internal/usagestats/aggregated.go (hunk 4 above)
Comment (efritz) [reply cmd/frontend/internal/usagestats/aggregated.go:127]: `event.LatenciesMonth` et al are what is calculated here, and search latencies are still calculated.

**Review by tjkandala — APPROVED** [review]
LGTM. re: the `prettier-check` build failure, it'll be easier to get the formatting right in your editor as prettier will format on save.

File: cmd/frontend/internal/usagestats/aggregated.go (hunk 4 above)
Comment (ebrodymoore) [reply cmd/frontend/internal/usagestats/aggregated.go:127]: Sooo I'm lost here so if we should remove more code somebody else should probably do it 😄

Comment (efritz) [reply cmd/frontend/internal/usagestats/aggregated.go:127]: I don't think you can remove anything else yet.

**Comment by ebrodymoore** [comment]
@dadlerj or @efritz I tried and did not succeed - I don't know how to resolve the Buildkite error I'm getting. Could one of you take it the rest of the way before branch cut?

**Pull request closed without merging**